src/tests/src/solver/modelParser/testModelParser.cpp

```diff
@@ -77,7 +77,7 @@ BOOST_AUTO_TEST_CASE(test_library_port_types)
                 - id: "porttype_id"
```

Comment:
Test **test_library_id_description** : 
considering the name of test, do last two checks 
```C++
 BOOST_CHECK(libraryObj.port_types.empty());
 BOOST_CHECK(libraryObj.models.empty());
```
relevant ?